InteractiveHtmlBom/Run.bat

```diff
@@ -1,35 +1,53 @@
 @echo off
+set pathofEDASourceFile=%1
+set FilePath=%~dp0
+
+::delete --show-dialog after frist start up and setting
+set option=--show-dialog
 
+::detect current language of user.    code of Chinese:0804(zh-cn),0404(zh-tw),1004(zh-sg),0C04(zh-hk)
+for /f "delims=" %%a in ('reg query "HKEY_LOCAL_MACHINE\SYSTEM\ControlSet001\Control\Nls\Language" /v InstallLanguage ^| findstr "0804"') do set language=%%a
```

Comment:
This registry entry stores installation language of windows. It may change later after install. To detect current preferred language you need to query HKCU\Control Panel\Desktop. Use install language if this fails.

More info here https://stackoverflow.com/a/35704694/4143646